kDrive/SceneDelegate.swift

```diff
@@ -55,26 +56,11 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate, AccountManagerDel
          The recommended approach is for the SceneDelegate to retain the scene's window.
      */
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
-        print(" scene session options")
+        Log.sceneDelegate("scene session options")
         /// 1. Capture the scene
```

Comment:
```suggestion
```

Reply:
Sure I'll stick to our standard which is "only protocols, or bugs and oddities"
I code this way, I'll make sure to match our rule before asking for review.

That said, now that we split the code in more SRP fashion, I do feel like we can get away with less comments.

Reply:
> it was part of a series of comments 1. 2. 3. (that I probably didn't notice in an earlier review) but now it's just sitting here.

For the record, this was part of my SceneDelegate PoC (the main PR) and was ought to be removed.
Is never was "upstreamed" to releaseAPIV3